README.md

```diff
@@ -5,11 +5,7 @@
 The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from a bash shell terminal, in the project root directory:
```

Comment:
You should probably mention that they need to install Vagrant (and a hypervisor)